pandas/tseries/frequencies.py

```diff
@@ -139,6 +139,12 @@ def infer_freq(index, warn: bool = True) -> Optional[str]:
         If the index is not datetime-like.
     ValueError
         If there are fewer than three values.
+
+    Examples
+    --------
+    >>> idx = pd.date_range(start = '2020/12/01', end = '2020/12/30', periods = 30)
```

Comment:
Could you eliminate the spaces between the `=` e.g `start = '2020/12/01'`

Reply:
@mroeschke done. I'm kinda newb to submitting my changes in public repos so I hope that I managed to do it correctly :)